docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,251 @@
+bin          vhbb_Zmm_gt400_13TeV
+observation  12.0
+--------------------------------------------------------------------------------
+bin                                   vhbb_Zmm_gt400_13TeV   vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV   vhbb_Zmm_gt400_13TeV     vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV vhbb_Zmm_gt400_13TeV
+process                               ggZH_lep_PTV_GT400_hbb ZH_lep_PTV_GT400_hbb ZH_lep_PTV_250_400_hbb ggZH_lep_PTV_250_400_hbb Zj1b            Zj0b_c          Zj0b_udsg       VVLF            Zj2b            VVHF
+process                               -3                     -2                   -1                     0                        1               2               3               4               5               6
+rate                                  0.0907733              0.668303             0.026293               0.00434588               3.78735         2.58885         4.09457         0.413716        7.02731         0.642605
+--------------------------------------------------------------------------------
+
+```
+
+where you can see the contributions from various background processes, namely Z+jets, $t\bar{t}$ and the single top, as well as signal processes corresponding to the STXS scheme discussed above. 
+
+One of the most important stages in the analysis design, is to make sure that the reco-level categories are pure with the corresponding gen-level processes.
+
+To explicitly check it, one can plot the contributions of gen-level bins in all of the reco-level bins. We propose to use the script provided in the tutorial git-lab page. 
+
+```shell
+python scripts/get_migration_matrix.py counting/combined_ratesOnly.txt
+
+```
+![](figures/migration_matrix_zh.png) 
+
+Now that we checked the response matrix we can attempt the maximum likelihood unfolding. We can use the `multiSignalModel` physics model available in `Combine`. To prepare the workspace we can run the following command: 
+
+```shell
+text2workspace.py -m 125  counting/* -P HiggsAnalysis.CombinedLimit.PhysicsModel:multiSignalModel  --PO verbose --PO 'map=.*/.*ZH_lep_PTV_75_150_hbb:r_zh_75_150[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_150_250_0J_hbb:r_zh_150_250noj[1,-5,5]'  --PO 'map=.*/.*ZH_lep_PTV_150_250_GE1J_hbb:r_zh_150_250wj[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_250_400_hbb:r_zh_250_400[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_GT400_hbb:r_zh_gt400[1,-5,5]' -o ws_counting.root
+```
+where we use `--PO 'map=bin/process:poi[init, min, max]'`. So in the example above a signal POI is assigned to each gen-level bin independent on reco-level bin. This allows to take into account the non-trivial acceptance effects. One can also perform bin-by-bin unfolding using the mapping to the bin names rather that processes, e.g. `'map= vhbb_Zmm_gt400_13TeV/.*:r_reco_zh_gt400[1,-5,5]'`, but this method is not recommended and can be used for tests.
+
+To extract the measurement let's run the initial fit first: 
```

Comment:
Maybe a few more words about `MultiDimFit`, e.g.

"We will use the MultiDimFit algorithm  to simultaneously extract best-fit values and uncertainties on multiple parameters."

Reply:
And maybe a short word about the initial values "we use `--setParameters <param>=<value>,...` to ... "